spec/order_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe Order do
+  let(:menu) do
+    double :menu,
+    show: [{dish: 'BBQ',       price: 10},
+           {dish: 'Sushi',     price: 6}]
+  end
+
+  it "has customer choose a dish and quantities" do
+    expect(subject.choose_dish(menu, 'BBQ', 3))
+      .to eq([{dish: 'BBQ', price: 10, quantities: 3}])
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)